cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -4369,6 +4381,159 @@ msgstr "Chwiliwch am gyfryngwr teulu yn eich ardal chi i ganfod a ydych yn gymwy
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Technical information about this service’s accessibility"
+msgstr "Gwybodaeth dechnegol am hygyrchedd y gwasanaeth hwn"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The Legal Aid Agency is committed to making its service accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (&#8470; 2) Accessibility Regulations 2018."
+msgstr "Mae’r Asiantaeth Cymorth Cyfreithiol wedi ymrwymo i wneud ei gwasanaeth yn hygyrch, yn unol â Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Rhaglenni Symudol) (Rhif 2) 2018."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Web Content Accessibility Guidelines version 2.1"
+msgstr "Canllawiau Hygyrchedd Cynnwys y We fersiwn 2.1"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "This service is partially compliant with the %(wcaglink)s AA standard, due to the non-compliance issues listed below."
+msgstr "Mae’r gwasanaeth hwn yn cydymffurfio’n rhannol â safon AA %(wcaglink)s, oherwydd y materion diffyg cydymffurfio a restrir isod."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Non-accessible content"
+msgstr "Cynnwys nad oes modd cael gafael arno"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "The content listed below is non-accessible for the following reasons."
+msgstr "Nid oes modd cael gafael ar y cynnwys a restrir isod am y rhesymau canlynol."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Non-compliance with the accessibility regulations"
+msgstr "Ddim yn cydymffurfio â’r rheoliadau hygyrchedd"
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Users are not able to turn off, adjust or extend the time in which inactivity will result in a loss of data. This fails WCAG 2.1 2.2.1 A (Timing Adjustable)."
+msgstr "Ni all defnyddwyr ddiffodd, addasu nac ymestyn yr amser pan fydd anweithgarwch yn arwain at golli data. Mae hyn yn methu WCAG 2.1 2.2.1 A (Addasu’r Amseriad)."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "Some page elements are not available in Welsh. This fails WCAG 2.1 3.1.1 A (Language of Page)."
+msgstr "Nid yw rhai elfennau tudalen ar gael yn Gymraeg. Mae hyn yn methu WCAG 2.1 3.1.1 A (Iaith Tudalen)."
+
+#: cla_public/templates/accessibility-statement.html:1
+msgid "When a user enters figures into form fields it is not clear that the field does not accept currency symbols. This fails WCAG 2.1 2.4.6 AA (Headings and Labels)."
+msgstr "Pan fydd defnyddiwr yn rhoi ffigurau mewn meysydd ffurflen, nid yw’n glir nad yw’r maes yn derbyn symbolau arian. Mae hyn yn methu WCAG 2.1 2.4.6 AA (Penawdau a Labeli)."
```

Comment:
These were removed from the accessibility statement?

Reply:
I queried the first with Josh and yes, it is to go.  

The second was one you fixed, so that can go too.

Reply:
Can we remove these completely from the translation file?

Reply:
We tend to keep old translations in the file.  Often commented out.  I'd leave them in for now at least.